pandas/tests/frame/indexing/test_indexing.py

```diff
@@ -1685,6 +1685,15 @@ def test_getitem_interval_index_partial_indexing(self):
         res = df.loc[:, 0.5]
         tm.assert_series_equal(res, expected)
 
+    @pytest.mark.parametrize("indexer", ["A", ["A"]])
```

Comment:
can you add the other 3 cases from OP that 'work fine' (2 x slice and a sorted case). ok to have another test if helpful.

Reply:
Done, worked through parametrization